engine/api/workermodel/dao.go

```diff
@@ -110,6 +142,100 @@ func LoadAllByNameAndGroupIDs(ctx context.Context, db gorp.SqlExecutor, name str
 	return getAll(ctx, db, query, opts...)
 }
 
+// LoadAllActiveAndNotDeprecatedForGroupIDs retrieves models for given group ids.
+func LoadAllActiveAndNotDeprecatedForGroupIDs(ctx context.Context, db gorp.SqlExecutor, groupIDs []int64, opts ...LoadOptionFunc) ([]sdk.Model, error) {
+	query := gorpmapping.NewQuery(`
+    SELECT *
+    FROM worker_model
+    WHERE group_id = ANY(string_to_array($1, ',')::int[])
```

Comment:
instead of  `WHERE group_id = ANY(string_to_array($1, ',')::int[])` you can do `WHERE group_id = ANY($1)`if you pass $1 as a `pq.Int64Array(ids)`